flow/flow.h

```diff
@@ -61,7 +61,8 @@
 #define TEST(condition)                                                                                                \
 	if (!(condition)) {                                                                                                \
 	} else {                                                                                                           \
-		static TraceEvent* __test = &(TraceEvent("CodeCoverage")                                                       \
+		Severity tracesev = intToSeverity(FLOW_KNOBS->TRACE_EVENT_SEVERITY);                                           \
+		static TraceEvent* __test = &(TraceEvent(tracesev, "CodeCoverage")                                             \
```

Comment:
```suggestion                                          \
		static TraceEvent* __test = &(TraceEvent(intToSeverity(FLOW_KNOBS->TRACE_EVENT_SEVERITY), "CodeCoverage")                                             \
```

`__test` is a function local static variable, so it's initialized only once when control first passes through its declaration. We want to move the call to intToSeverity inside the expression where we're initializing `__test`, so that we only call intToSeverity once.